training/decommission-nodes.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Decommission Nodes
+summary: Learn how to remove nodes from your cluster.
+toc: false
+sidebar_data: sidebar-data-training.json
+---
+
+<div id="toc"></div>
+
+## Presentation
+
+<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vR0C9G-W0sNE1-SwziVHU7jJikwx50K-cDwy0kdFkADqSeu1po_nq8OM-IHyqngaeKtRiUNjKcZXZJL/embed?start=false&loop=false" frameborder="0" width="756" height="454" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
+
+#### URL for comments
+
+[Decommission Nodes](https://docs.google.com/presentation/d/1beTO2-Ca7Wg1TDSHMU-J-tdEJJuwmaPIm6KGWufX7jw/edit)
+
+## Lab
+
+In this lab, we'll walk you through decommissioning a node, which safely removes it from your cluster.
+
+### Before You Begin
+
+To complete this lab, you should have a [local cluster of 3 nodes](3-node-local-secure-cluster.html).
+
+### Step 1. Scale the Cluster
+
+Because our cluster has a replication factor of 3, we must add a fourth node to ensure we have enough nodes to replicate our data.
```

Comment:
Sentence makes no sense - three nodes is enough. The fourth node would be added once three nodes aren't enough (or we want to make sure data can upreplicate should one node fail).

Reply:
Btw, in the preview I saw there's this which also isn't quite correct (but I think you've just updated this since, just posting to make sure)

![image](https://user-images.githubusercontent.com/5076964/32641983-292d8ab2-c59f-11e7-8256-86383b1a9c73.png)

Reply:
This is in context of decommissioning a node. We have to add the fourth node because we're going to take one down, which means we'll only have 3 at the end of the process. I'll make this more clear in the doc, though.